components/lib/treeselect/TreeSelectBase.js

```diff
@@ -124,9 +124,8 @@ const styles = `
 }
 
 .p-treeselect-clear-icon {
-    position: absolute;
     top: 50%;
-    margin-top: -.5rem;
+    margin-top: 11px;
```

Comment:
PrimeTek usually doesn't use Px they use rem units so can you tweak it with rem units?